marketplace/README.md

```diff
@@ -48,3 +48,16 @@ A repository can be validated using the `MarketplaceClient`. The client pulls a
         e.printStackTrace();
     }
 ```
+
+Note that in this example we have used the default `SignatureType.BASE64_ENCODED` which specifies that the signature files are
+base64 encoded. If you require non-base64 encoded use `SignatureType.SIG`.
```

Comment:
Weird line space wrapping here and below?

Reply:
think its fine